### PR TITLE
gstreamermm 1.8.0

### PR DESCRIPTION
**Pull request by tschoonj**

### CODE REVIEW

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

File: Formula/gstreamermm.rb [hunk 1]
```diff
@@ -20,6 +19,12 @@ class Gstreamermm < Formula
 
   needs :cxx11
 
+  # Bug filed upstream with patch at https://bugzilla.gnome.org/show_bug.cgi?id=772645
```
Comment (ilovezfs) [inline Formula/gstreamermm.rb]: Necessary, I don't know about that. But it is customary. Here are some examples:
https://gist.github.com/ilovezfs/62a9eb715dc044f7202cbf86caaf713b

**Comment by tschoonj** [comment]
@ilovezfs is there a reason why this one didn't get closed automatically yesterday?

**Comment by ilovezfs** [comment]
Commit message was missing https://github.com/Homebrew/homebrew-core/commit/7373a619a0cb73b70281dfa3b7420823bb62dbf0

**Comment by tschoonj** [comment]
So I can close it myself?

**Comment by ilovezfs** [comment]
@tschoonj it was my fault. You have to use "squash" not "fixup" because the stuff `brew pull` adds to the commit messages goes in the last commit not the first commit.

**Comment by ilovezfs** [comment]
Yup, you can close it.

**Comment by tschoonj** [comment]
OK, thanks will keep it in mind

**Pull request closed without merging**